.github/workflows/upload-contrib-stats.yml

```diff
@@ -0,0 +1,36 @@
+name: Upload test stats
+
+on:
+  schedule:
+    # Choose a random time near midnight PST because it may be delayed if there are high loads
+    - cron:  37 7 * * *
+
+jobs:
+
+  upload-contribution-stats:
+    steps:
+      - name: Checkout PyTorch
+        uses: pytorch/pytorch/.github/actions/checkout-pytorch@master
+      - run: |
+          pip3 install requests==2.26
+          pip3 install rockset==1.0.3
+          pip3 install boto3==1.19.12
+
+      - name: Upload external contribution stats
+        env:
+          ROCKSET_API_KEY: ${{ secrets.ROCKSET_API_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "Uploading external contribution stats for $(date -v-1d +%F)"
+          python3 -m tools.stats.upload_external_contrib_stats --startDate $(date -v-1d +%F)  
+
+  check-api-rate:
```

Comment:
You don't need this here I think. This was mainly used to debug rate limit we have with GitHub a while back (then GitHub increased our limit, and the issue seems to go away for now)